### PR TITLE
manage classification.config

### PR DESCRIPTION
**Pull request by inashivb**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/3203

### CODE REVIEW

File: suricata/update/main.py [hunk 1]
```diff
@@ -357,6 +357,56 @@ def load_dist_rules(files):
...
+            with open(path) as fp:
+                for line in fp:
+                    if line.startswith("#") or not line.strip():
+                        continue
+                    config_classification = line.split(":")[1].strip()
+                    key, desc, priority = config_classification.split(",")
+                    if key in classification_arr:
+                        if classification_arr[key][1] >= priority:
+                            continue
+                    classification_arr[key] = [desc, priority, line.strip()]
```
Comment (jasonish) [inline suricata/update/main.py:380]: Some debug logging line is needed here.  When I run with `-v` I should be able to see what classification.config its loading off my file system.

**Comment by inashivb** [comment]
> We have #221 to help address this. But I was thinking store the loaded files in a list instead of a dict might be a more straightforward way to handle this.
> 
Do you mean a list of classification files? Would this work? https://github.com/OISF/suricata-update/pull/236/files#diff-b117dc9956be2a723a248660048c4a4bR1080

**Comment by jasonish** [comment]
> Do you mean a list of classification files? Would this work? https://github.com/OISF/suricata-update/pull/236/files#diff-b117dc9956be2a723a248660048c4a4bR1080

Was thinking a list for all files. Not sure if that would work, cause its working on the "dict" of files loaded right? In the case, duplicate files by name (not content) are already lost.

**Comment by jasonish** [comment]
> Was thinking a list for all files. Not sure if that would work, cause its working on the "dict" of files loaded right? In the case, duplicate files by name (not content) are already lost.

Outside of the scope of this PR though.

**Comment by inashivb** [comment]
> > Do you mean a list of classification files? Would this work? https://github.com/OISF/suricata-update/pull/236/files#diff-b117dc9956be2a723a248660048c4a4bR1080
> 
> Was thinking a list for all files. Not sure if that would work, cause its working on the "dict" of files loaded right? In the case, duplicate files by name (not content) are already lost.

We could use hash of the files as keys instead of the name. Would that make sense?

**Comment by jasonish** [comment]
> We could use hash of the files as keys instead of the name. Would that make sense?

Yeah, it might.  I have to jump back into #221 and solve this issue for good.

**Comment by inashivb** [comment]
Replaced with #239

**Pull request closed without merging**